### ISSUE TITLE
 replace the default model 

### ISSUE THREAD

**Issue by R0FGV** [alphacep/vosk-api#267]
I want to know how to replace the default model with the model in Kaldi? For example: https://kaldi-asr.org/models/m2 What should I do specifically, is there a related tutorial?

**Comment by nshmyrev** [comment]
CVTE model uses pitch and requires code modification.

Otherwise you have to copy the files, as in https://alphacephei.com/vosk/models#model-structure

**Comment by R0FGV** [comment]
I found a short video from youtube to test generating subtitles. But compared with Google's speech recognition api (Google's on the left), I found that the segmentation is not very good, and there is no segmentation when it is needed. How can I solve this problem myself and make it closer to Google's segmentation?
image
![image](https://user-images.githubusercontent.com/19688305/97954260-bf124680-1dee-11eb-8455-165da01dd9cb.png)

**Comment by R0FGV** [comment]
> 
> 
> CVTE model uses pitch and requires code modification.
> 
> Otherwise you have to copy the files, as in https://alphacephei.com/vosk/models#model-structure

I found a short video from youtube to test generating subtitles. But compared with Google's speech recognition api (Google's on the left), I found that the segmentation is not very good, and there is no segmentation when it is needed. How can I solve this problem myself and make it closer to Google's segmentation?
![image](https://user-images.githubusercontent.com/19688305/97954326-eec14e80-1dee-11eb-87af-40a39e1af2b7.png)
